modules/installing-oadp-aws-sts.adoc

```diff
@@ -0,0 +1,253 @@
+
+.. If you are using only CSI volumes, deploy a Data Protection Application by entering the following command:
++
+[source,terminal]
+----
+$ cat << EOF | oc create -f -
+  apiVersion: oadp.openshift.io/v1alpha1
+  kind: DataProtectionApplication
+  metadata:
+    name: ${CLUSTER_NAME}-dpa
+    namespace: openshift-adp
+  spec:
+    backupImages: true <1>
+    features:
+      dataMover:
+        enable: false
+    backupLocations:
+    - bucket:
+        cloudStorageRef:
+          name: ${CLUSTER_NAME}-oadp
+        credential:
+          key: credentials
+          name: cloud-credentials
+        prefix: velero
+        default: true
+        config:
+          region: ${REGION}
+    configuration:
+      velero:
+        defaultPlugins:
+        - openshift
+        - aws
+        - csi
+      restic:
+        enable: false
+EOF
+----
+<1> Set this field to `false` if you do not want to use image backup.
+
+// . Create the `DataProtectionApplication` resource, which is used to configure the connection to the storage where the backups and volume snapshots are stored:
+
+.. If you are using CSI or non-CSI volumes, deploy a Data Protection Application by entering the following command:
++
+[source,terminal]
+----
+$ cat << EOF | oc create -f -
+  apiVersion: oadp.openshift.io/v1alpha1
+  kind: DataProtectionApplication
+  metadata:
+    name: ${CLUSTER_NAME}-dpa
+    namespace: openshift-adp
+  spec:
+    backupImages: true <1>
+    features:
+      dataMover:
+         enable: false
+    backupLocations:
+    - bucket:
+        cloudStorageRef:
+          name: ${CLUSTER_NAME}-oadp
+        credential:
+          key: credentials
+          name: cloud-credentials
+        prefix: velero
+        default: true
+        config:
+          region: ${REGION}
+    configuration:
+      velero:
+        defaultPlugins:
+        - openshift
+        - aws
+      nodeAgent: <2>
+        enable: false
+        uploaderType: restic
+    snapshotLocations:
+      - velero:
+          config:
+            credentialsFile: /tmp/credentials/openshift-adp/cloud-credentials-credentials <3>
+            enableSharedConfig: "true" <4>
+            profile: default <5>
+            region: ${REGION} <6>
+          provider: aws
+EOF
+----
+<1> Set this field to false if you do not want to use image backup.
+<2> See the first note regarding the `nodeAgent` attribute.
+<3> The `credentialsFile` field is the mounted location of the bucket credential on the pod.
+<4> The `enableSharedConfig` field allows the `snapshotLocations` to share or reuse the credential defined for the bucket.
+<5> Use the profile name set in the {aws-short} credentials file.
+<6> Specify `region` as your {aws-short} region. This must be the same as the cluster region.
++
+You are now ready to back up and restore {product-title} applications, as described in _Backing up applications_.
+
+[NOTE]
```

Comment:
These two admonitions need reworked, as we can't have them in "close proxomity to one another": https://redhat-documentation.github.io/supplementary-style-guide/#admonitions

Reply:
I have opened a ticket for this as a doc improvement. It needs to be done elsewhere in the ROSA documentation which was recently published. Can we please do this after this is published?

https://access.redhat.com/documentation/en-us/openshift_container_platform/4.15/html/backup_and_restore/oadp-application-backup-and-restore#oadp-rosa-backing-up-applications

https://issues.redhat.com/browse/OADP-3768